src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.command.Command;
+
+/**
+ * A chatbot that helps manages your tasks.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Class constructor. Initialises a Duke instance with the Ui, TaskList, and Storage.
+     *
+     * @param filePath
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            taskList = new TaskList(storage.load());
+        } catch (DukeException e) {
+            ui.showLoadingError();
+            taskList = new TaskList();
+        }
+    }
+
+    /**
+     * Empty class constructor.
+     */
+    public Duke() {
+        this("src/main/data/dukeSave.txt");
+    }
+
+    /**
+     * To run the program.
+     */
+    public void run() {
+        ui.showStartup();
+        boolean isExit = false;
+        while (!isExit) {
+            try {
+                String fullCommand = ui.readCommand();
+                Command c = Parser.parseInput(fullCommand);
+                c.runCommand(taskList, ui, storage);
+                isExit = c.isExit();
+            } catch (DukeException e) {
+                ui.showError(e.getMessage());
+            } finally {
+                ui.showLine();
+            }
+        }
+
+    }
+
+    /**
+     * This is the main method.
+     *
+     * @param args Unused.
+     */
+    public static void main(String[] args) {
+        new Duke("src/main/data/duke.txt").run();
+    }
+
+    /**
+     * You should have your own function to generate a response to user input.
```

Comment:
Maybe you could start your comment on this method with a Verb? Example for this would be 
/**
 Returns a response ....